packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -95,6 +107,12 @@ class BottomSheet extends StatefulWidget {
   /// Default is true.
   final bool enableDrag;
 
+  /// Called when the user begins dragging the bottom sheet vertically.
+  final BottomSheetDragStartHandler onDragStart;
```

Comment:
The new callbacks should explain how they'd typically be used and how they relate to the existing onClosing and enableDrag parameters.

Reply:
Done